pay-api/migrations/versions/15682073c535_account_credits.py

```diff
@@ -0,0 +1,52 @@
+"""account_credits
+
+Revision ID: 15682073c535
+Revises: 331c2cdb0b94
+Create Date: 2020-12-16 12:25:50.774792
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '15682073c535'
+down_revision = '331c2cdb0b94'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('credits',
+                    sa.Column('id', sa.Integer(), autoincrement=True, nullable=False),
+                    sa.Column('cfs_identifier', sa.String(length=50), nullable=True),
+                    sa.Column('is_credit_memo', sa.Boolean(), nullable=True),
+                    sa.Column('amount', sa.Float(), nullable=False),
+                    sa.Column('remaining_amount', sa.Float(), nullable=False),
+                    sa.Column('account_id', sa.Integer(), nullable=True),
+                    sa.ForeignKeyConstraint(['account_id'], ['payment_account.id'], ),
+                    sa.PrimaryKeyConstraint('id')
+                    )
+    op.create_index(op.f('ix_credits_account_id'), 'credits', ['account_id'], unique=False)
+    op.create_index(op.f('ix_credits_cfs_identifier'), 'credits', ['cfs_identifier'], unique=False)
+    op.drop_column('payment', 'created_on')
+    op.drop_column('payment_account', 'running_balance')
```

Comment:
the running_balance removal is intentional rite? i guess there is no data in prod to migrate

Reply:
Yea, not needed at this moment and was never populated. Removed it to avoid any confusion.